### ISSUE TITLE
Can't compile with latest cocoalumberjack

### ISSUE THREAD

**Issue by ggb88** [robbiehanson/CocoaAsyncSocket#389]
Have CocoaAsyncSocket 7.4.2 / CocoaLumberjack 2.2.0 via pods.  Have enabled in GCDAsyncSocket

```
 #define GCDAsyncSocketLoggingEnabled 1
```

Unfortunately can no longer can compile.  Get "Implicit declaration of function "LOG_C_MAYBE" is invalid in C99" Have tried enabling legacy macros in LJ, but that made it worse.  Am I even doing this right?

**Comment by ggb88** [comment]
I'm getting nowhere with this .. hope someone can help.  I've tried debugging this and changing things around and now get linker errors on _DDExtractFileNameWithoutExtension.  A little more detail on my environment.  Running Xcode7.2 / deploy target 9.2 - Created a single view template app for iOS and included 2 pods - cocoa async socket and cocoa lumberjack (latest versions).  Change the debugging enabled flag as noted above.  All I got for x-mas this year is one step closer to insanity :)
